src/components/Hotspot.tsx

```diff
@@ -217,14 +182,13 @@ export default class Hotspot extends Component<Props, State> {
       width: layout.width
     };
 
-    const fontSizeToUse = `${this.getFontSize(layout, hotspot, label || '')}px`;
-
     const buttonStyles = {
       ...defaultButtonsStyles,
       ...hotspot.styles,
       cursor: disableClick ? 'default' : 'pointer',
       maxWidth: `${layout.width}px`,
-      fontSize: fontSizeToUse
+      fontSize: `${hotspot.relativeStyle.fontSize}px`,
```

Comment:
are these values always set ?

Reply:
this is build in the code so yes

Reply:
i mean
layout.width is always set ?
hotspot.relativeStyle.fontSize is always set ?